lib/logstash/inputs/stomp.rb

```diff
@@ -37,6 +37,12 @@ class LogStash::Inputs::Stomp < LogStash::Inputs::Base
   #Auto reconnect interval in seconds
   config :reconnect_interval, :validate => :number, :default => 30
 
+  # Enable TLS/SSL connection?
+  config :ssl, :validate => :boolean, :default => false
```

Comment:
Trusted certificate authorities are supported by the underlying Java cacerts file.  Overriding the standard truststore, if needed, can be accomplished by setting the Java system property javax.net.ssl.trustStore.